File: Kody/Problem2/problem.java

public class problem{
	public static void main(String[] args){
		int sum = 0;
		int fib1 = 1, fib2 = 0, fib3 = 0;
		do{
			fib3 = fib1 + fib2;
			if(fib3 %2 == 0){
				sum += fib3;
			}
			fib2 = fib1;
			fib1 = fib3;
		}while(fib3 < 4000000);
		System.out.println(sum);
	}
}

File: Kody/Problem2/problem.cpp
#include <stdio.h>
int main(){
	long unsigned int sum = 0;
	long unsigned int fib1 = 1, fib2 = 0, fib3 = 0;
	do{
		fib3 = fib2 + fib1;
		if(fib3 > 4000000){
			break;
		}
		printf("(%lu, %lu, %lu)", fib2, fib1, fib3);
		fib2 = fib1;
		fib1 = fib3;
		if(fib3 % 2 == 0){
			printf("+");
			sum += fib3;
		}
		printf("\n");
	}while(fib3 < 4000000);
	printf("\n%lu\n", sum);
}
